extensions/wikia/FeedsAndPosts/JustUpdatedAPIProxy.class.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Wikia\FeedsAndPosts;
+
+class JustUpdatedAPIProxy {
+
+	const LIMIT = 4;
+
+	private function requestAPI($params) {
+		$api = new \ApiMain(new \FauxRequest($params));
+		$api->execute();
+
+		return $api->GetResultData();
+	}
+
+	private function getRecentChanges() {
+		$response = $this->requestAPI( [
+			'action' => 'query',
+			'list' => 'recentchanges',
+			'rcprop' => 'title',
+			'rcshow' => '!bot|!minor|!redirect',
+			'rcnamespace' => '0',
```

Comment:
please use $wgContentNamespaces instead of hardcoded 0, some wikis have their custom namespaces that are used as regular articles. To get results for multiple namespaces just separate them with `|`